config/software/ruby-windows.rb

```diff
@@ -25,29 +24,43 @@
     relative_path "ruby-#{version}-i386-mingw32"
     source url: "https://s3-us-west-2.amazonaws.com/yakyakyak/ruby-#{version}-i386-mingw32.7z"
 
+    default_version "2.0.0-p647"
     version("2.0.0-p647") { source md5: "0b1e8f16580f26fd0992fad3834cb83d" }
   end
 elsif windows_arch_i386?
+  default_version "2.0.0-p648"
```

Comment:
I'm cool with sending that through next, wanted to stick with major/minor for this one to reduce the widespread risk.